src/cpp/tests/automation/testthat/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.save
```

Comment:
I often rename some of the test files to *.R.save to run a subset of tests, .git-ignoring them somewhat cuts down the clutter in the VSCode git UI.

Reply:
Sounds good!